jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/SyncServerlessAppAction.kt

```diff
@@ -97,20 +97,18 @@ class SyncServerlessAppAction(private val codeOnly: Boolean = false) : AnAction(
             }
 
             val templateFunctions = SamTemplateUtils.findFunctionsFromTemplate(project, templateFile)
-            val hasImageFunctions = templateFunctions.any { (it as? SamFunction)?.packageType() == PackageType.IMAGE }
+            val hasImageFunctions: Boolean = templateFunctions.any { (it as? SamFunction)?.packageType() == PackageType.IMAGE }
```

Comment:
why did this come back?

Reply:
Keeping this as required to avoid any conflicts due to null